src/app/app.component.ts

```diff
@@ -7,4 +11,25 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'poll';
+  private apiUrl = 'http://localhost:3000/api/questions';
+  data: any = {};
```

Comment:
you should never use any, please provide type

Reply:
string